### PR TITLE
docs: improve the guide section

### PR DESCRIPTION
**Pull request by nico-bachner**
same deal as #3 

Hope I didn't make any mistakes of my own.

### CODE REVIEW

File: docs/guide/exporting.md [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
 
-> Exporting is relying on [Playwright](https://playwright.dev) to do the rendering, you will need to install [`playwright-chromium`](https://playwright.dev/docs/installation#download-single-browser-binary) for using this feature.
-> If you are doing exporting in a CI environment, [this guide](https://playwright.dev/docs/ci) could also be helpful.
+> Exporting relies on [Playwright](https://playwright.dev) to do the rendering. You will therefore need to install [`playwright-chromium`](https://playwright.dev/docs/installation#download-single-browser-binary) to use this feature.
+> If you are doing exporting in a CI environment, [the playwright CI guide](https://playwright.dev/docs/ci) can be helpful.
 
 ## PDF
 
-Exporting your slides into PDF by the following command
+Install `playwright-chromium` if you haven't already
```
Comment (antfu) [inline docs/guide/exporting.md:8]:
```suggestion
Install `playwright-chromium`
```

File: docs/guide/syntax.md [hunk 2]
```diff
@@ -64,15 +64,15 @@ Refer to [customization](/custom/) for more details.
 
 ## Code Snippet
 
-A big reason I am making this is that I need to make my code looks just right in the slides. So just as you expected, you can use Markdown favored code block to highlight your code.
+A big reason I am building Slidev is needing to make my code look just right in the slides. So just as you expected, you can use Markdown flavored code block to highlight your code.
 
-~~~ts
+~~~md
```
Comment (antfu) [inline docs/guide/syntax.md:69]: Ah, this is on purpose to fool Prism to get proper highlighting, Can you revert them all? Thanks.

Comment (nico-bachner) [reply docs/guide/syntax.md:69]: sure thing

**Comment by nico-bachner** [comment]
Not sure if I left one out though

**Comment by antfu** [comment]
You are on fire, thanks a lot!

**Comment by nico-bachner** [comment]
Absolutely no problem

**Pull request merged**